actors/builtin/paych/paych_test.go

```diff
@@ -124,41 +124,47 @@ func TestPaymentChannelActor_CreateLane(t *testing.T) {
 		nonce uint64
 		amt   int64
 
+		paymentChannel addr.Address
+
 		secretPreimage []byte
 		sig            *crypto.Signature
 		verifySig      bool
 		expExitCode    exitcode.ExitCode
 	}{
 		{desc: "succeeds", targetCode: builtin.AccountActorCodeID,
-			amt: 1, epoch: 1, tlmin: 1, tlmax: 0,
+			amt: 1, paymentChannel: paychAddr, epoch: 1, tlmin: 1, tlmax: 0,
 			sig: sig, verifySig: true,
 			expExitCode: exitcode.Ok},
+		{desc: "succeeds", targetCode: builtin.AccountActorCodeID,
```

Comment:
Desc should be "fails if channel address doesn't match"

Reply:
This is done.